js/bootstrap-slider.js

```diff
@@ -552,14 +553,25 @@
 				this.mousePos = 'pageX';
 				this.sizePos = 'offsetWidth';
 
-				this._addClass(this.tooltip, 'top');
-				this.tooltip.style.top = -this.tooltip.outerHeight - 14 + 'px';
+				if(this.options.tooltip_position === 'bottom'){
+					this._addClass(this.tooltip, 'bottom');
```

Comment:
could you refactor this into a private method?

it could take the tooltip object as an argument and do the following:

- add the 'bottom' class to the tooltip
- set the style top offset